neofs-local/HOWTO.md

```diff
@@ -155,6 +157,9 @@ drwxr-xr-x    1 root     root        4.0K Sep  2 10:04 ..
 
 ## Let's upload my cat's foto 
 
+**Attention** without creation of StorageGroup, your files will be removed in a few minutes.
+How to create StorageGroup you can find [here](##Let's create StorageGroup for our files).
```

Comment:
copy this line to 203/208 (`Let's upload my cat's video`)

Reply:
done